contributors.json

```diff
@@ -1,6 +1,20 @@
 {
 "contributors": [
 { "profile_name":"ajdawson", "organisation":""},
+{ "profile_name":"rhattersley", "organisation":"Met Office"},
+{ "profile_name":"pelson", "organisation":"Met Office"},
```

Comment:
It is a reasonable state - they are CLAs for two distinct organisations.
INAL but: I think that covers section 4 of the CLA:


> 4. You represent that you are legally entitled to grant the above licence. If your employer(s) has rights to intellectual property that you create that includes your Contributions, you represent that you have received permission to make Contributions on behalf of that employer, that your employer has waived such rights for your Contributions to the Met Office.